subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -1541,6 +1541,11 @@ u16_t ull_conn_lll_max_tx_octets_get(struct lll_conn *lll)
 	return max_tx_octets;
 }
 
+void ull_conn_upd_curr_reset(void)
```

Comment:
Please replace locations where conn_upd_curr is reset with a call to this function